Libraries/LibJS/Tests/automatic-semicolon-insertion.js

```diff
@@ -1,30 +1,26 @@
-load("test-common.js");
+test("Issue #1829, if-else without braces or semicolons", () => {
+    const source = 
+`if (1)
+    return 1;
+else
+    return 0;
 
-/**
- * This file tests automatic semicolon insertion rules.
- * If this file produces syntax errors, something is wrong.
- */
+if (1)
+    return 1
+else
+    return 0
 
-function bar() {
-    // https://github.com/SerenityOS/serenity/issues/1829
-    if (1)
-        return 1;
-    else
-        return 0;
+if (1)
+    return 1
+else
+    return 0;`;
 
-    if (1)
-        return 1
-    else
-        return 0
+    expect(source).toCompile();
```

Comment:
This is a great API, but I think the naming can be improved, as we're not really "compiling" anything - maybe `toParseWithoutErrors()`?